osi_featuredata.proto

```diff
@@ -44,35 +49,35 @@ message DetectionHeader
     //
     optional uint32 cycle_counter = 3;
 
-    // Mounting position of the sensor (origin and orientation of the sensor
-    // coordinate system); given relative to the middle of the rear axle of the
-    // host vehicle.
+    // Mounting position of the sensor (origin and orientation of the cartesian
+    // sensor coordinate system); given relative from the middle of the rear
+    // axle of the host vehicle coordinate system (see: \c OSI::Vehicle vehicle
+    // reference point) to the sensor detection coordinate system's origin. The
+    // sensor detection coordinate system is a spherical polar coordinate system
+    // (same origin and orientation as cartesian sensor coordinate system).
     //
-    // The x-axis must be the angle bisector of the horizontal field of view in
-    // a right-handed coordinate system. The xy-plane represents the
-    // horizontal field of view and the xz-plane the vertical field of view.
+    // The sensor detection coordinate system's x-axis must be the angle
+    // bisector of the sensor's horizontal and vertical field of view in a
+    // right-handed coordinate system. The xy-plane represents the horizontal
+    // field of view and the xz-plane the vertical field of view.
```

Comment:
Do we have to define a cartesian AND a spherical sensor frame? (I would stick to the word "frame" as in OSI:Spherical3d and not use "coordinate system" here. Would be nice to have a consistent wording throughout OSI.) Furthermore, I would recommend clarifying the meaning of "horitzontal" and "vertical" as misunderstandings might arise otherwise. I propose the following formulation:


``Mounting position of the sensor (origin and orientation of the sensor frame). Both origin and orientation are given in and with respect to the host vehicle coordinate system (see: \c OSI::Vehicle vehicle reference point).``

``The sensor frame's x-axis is pointing in the central viewing direction of the sensor. It is the angle bisector of the sensor's horizontal and vertical field of view. The terms horizontal and vertical must be understood as names for the two principal axes of the sensor's field of view, which do not have to be horizontal or vertical in the strict sense of being parallel or perpendicular to the local gravitational vector. The horizontal field of view defines the sensor frame's xy-plane and the vertical field of view defines the xz-plane. The sensor frame is right-handed and the z-axis is pointing in an upward direction.``

Reply:
The sensor frame is cartesian. But detections may be defined with spherical coordinates. To distinguish this possible change I would propose to use "sensor frame" (cartesian) and "sensor detection frame" (spherical, cartesian, ...).

Horizontal and vertical is defined relative to the sensor (detection) frame's orientation.